website/content/api-docs/system/internal-counters.mdx

```diff
@@ -90,22 +90,6 @@ $ curl \
 This endpoint returns client activity information for a given billing
 period, which is represented by the `start_time` and `end_time` parameters.
 
-There are a few things to keep in mind while using this API.
-
-- For Vault versions before 1.11, the activity information only accounts for
-the activity of the latest contiguous months in the billing period.
-For example, if the billing period is from Jan 2022 to June 2022, and the
-activity subsystem in Vault was not capturing data for the months Jan to March,
-the response will only include information for May and June. Note that if no
-`start_time` and `end_time` parameters are specified, the billing period defaults
```

Comment:
Deleted this start_time/end_time information because it duplicates info that's there when we explain the parameters at the bottom with the API.